src/jsc/jsc_function.hpp

```diff
@@ -29,6 +29,17 @@ inline JSValueRef jsc::Function::call(JSContextRef ctx, const JSObjectRef& funct
 {
     JSValueRef exception = nullptr;
     JSValueRef result = JSObjectCallAsFunction(ctx, function, this_object, argc, arguments, &exception);
+
+    // Flush the React Native UI task queue whenever we call into JS from C++ – see `_flushUiTaskQueue` in
+    // `lib/extensions.js` for detailed explanation of why this is necessary.
+    JSObjectRef global_object = JSContextGetGlobalObject(ctx);
+    JSObjectRef realm_constructor = jsc::Object::validated_get_constructor(ctx, global_object, "Realm");
+    JSObjectRef flush_ui_task_queue =
+        jsc::Object::validated_get_function(ctx, realm_constructor, "_flushUiTaskQueue");
+    JSValueRef flush_ui_task_queue_exception = nullptr;
```

Comment:
I wonder if we would want to throw this `flush_ui_task_queue_exception` if it does fail?